src/vcModals.cpp

```diff
@@ -629,7 +629,40 @@ void vcModals_DrawProfile(vcState* pProgramState)
 
     ImGui::EndPopup();
   }
+}
+
+void vcModals_DrawConvert(vcState* pProgramState)
+{
+  const char *pModalName = udTempStr("%s###convertDock", vcString::Get("convertTitle"));
+  
+  if (pProgramState->openModals & (1 << vcMT_Convert))
+  {
+    ImGui::OpenPopup(pModalName);
+    ImGui::SetNextWindowSize(ImVec2(900, 600), ImGuiCond_FirstUseEver);
+  }
+
+  if (ImGui::BeginPopupModal(pModalName))
+  {
+    pProgramState->modalOpen = true;
+
+    ImGui::Columns(2, NULL, false);
+    ImGui::SetColumnWidth(0, ImGui::GetWindowSize().x - 125.f);
+
+    ImGui::TextUnformatted(vcString::Get("convertTitle"));
 
+    ImGui::NextColumn();
+    if (ImGui::Button(vcString::Get("popupClose"), ImVec2(-1, 0)) || vcHotkey::IsPressed(vcB_Cancel))
+      ImGui::CloseCurrentPopup();
+
+    ImGui::EndColumns();
+    ImGui::Separator();
+
+    if (ImGui::BeginChild("__convertPane"))
```

Comment:
Is this a standard naming convention in ImGui or something?

Reply:
Not as far as I'm aware.